### DISCUSSION TITLE
Getting group's "active" status using dBus

### DISCUSSION THREAD

**Discussion by exquo** [AsamK/signal-cli#975] [Q&A] [answered]
In the output of `signal-cli listGroups` there is an `Active: true` field for each group. But it is absent from the output of the [dBus version](https://github.com/AsamK/signal-cli/blob/9f7979314f2e3d5338b704f43e81da5a408d97b9/man/signal-cli-dbus.5.adoc#group-related-methods) of the `listGroups` command.
How can this status be obtained from a dbus method? 

The man page lists [a number of properties](https://github.com/AsamK/signal-cli/blob/9f7979314f2e3d5338b704f43e81da5a408d97b9/man/signal-cli-dbus.5.adoc#signalgroup-interface) that can be obtained through a group interface, including  

	IsMember (read-only) : always true (object path exists only for group members)

But there does not seem to be a counterpart to the plain-output's "active" property.

Also, what exactly does an "active" status signify? Is a "non-active" group a group that I am no longer a member of? If so, should such group be even included in the groups' listing?
Somewhat related? #638

**Comment by AsamK** [comment, marked as answer]
`Active: true` means the user is a member of the group (the name comes from the group sync message which has an `active` field)
So the `IsMember` dbus property corresponds to the active flag.
The listGroups command outputs all groups. When the user leaves a group the group information doesn't get deleted automatically, which matches the Signal-Android behavior.

**Reply by exquo** [reply]
Thanks, that clarifies it!
